### DISCUSSION TITLE
I could not possibly pull this off without Tweakpane

### DISCUSSION THREAD

**Discussion by dinther** [cocopon/tweakpane#635] [General]
![image](https://github.com/user-attachments/assets/ec70d352-79bd-4f42-8490-0706f0533f22)

This is a screenshot of my Beatline Beam Editor. Completely browser based software served from static files (I am weird like that). this software is used to create laser shows.

I have been working with Tweakpane for quite a while now. And I find it can do everything I throw at it. But at the cost of countless hours stuffing around and every time being surprised that the solution isn't that hard.

In short, documentation can do with some work. I am sure there are a lot of people using Tweakpane but when I search for answers online it is thin pickings. Probably an indication that I am thick and nobody else has those issues.

Anyway, I want to avoid to moan, I am here to say Thank you for writing Tweak Pane and making it beautiful out of the box.

**Comment by cocopon** [comment]
Thank you for using Tweakpane heavily, and great work on your project.

If you have specific ideas or suggestions for improving the documentation, please feel free to share them.
